core/src/trezor/ui/model/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+This module contains constants and functions that are different across
+hardware, e.g. T1 and TT.
+"""
+
+from .. import WIDTH
+
+IS_T1 = WIDTH == 128
```

Comment:
let's expose a constant in `modtrezorutils` for this. probably model string? enum?

Reply:
There is already such constant => `trezor.utils.MODEL`

Reply:
Fixed 0f45a44393684db76692ee4ea8ea14682ac89216.